platform/iotex/api.go

```diff
@@ -25,10 +24,37 @@ func (p *Platform) Coin() coin.Coin {
 	return coin.Coins[coin.IOTX]
 }
 
+func (p *Platform) CurrentBlockNumber() (int64, error) {
+	return p.client.GetLatestBlock()
+}
+
+func (p *Platform) GetBlockByNumber(num int64) (*blockatlas.Block, error) {
+	var normalized []blockatlas.Tx
+	txs, err := p.client.GetTxsInBlock(num)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, action := range txs {
+		tx := Normalize(action)
```

Comment:
Can you use go funk lib to do filter? Might be easier

Reply:
The funk lib doesn't look good, prefer just for loop